src/visualization/types/Map/view.tsx

```diff
@@ -126,16 +124,23 @@ const GeoPlot: FC<Props> = ({result, properties}) => {
         colorField: '_value',
         colors: DEFAULT_THRESHOLDS_GEO_COLORS,
         isClustered: false,
-        tooltipColumns: tooltipColumns,
+        tooltipColumns: [],
```

Comment:
I think this should be updated to be:
```
tooltipColumns: [],
colors: []
```
because we want to to give them a default value but later we can check if `layerOpts[0].tooltipColumns` is empty, then we can updated to use fluxgroup key union values.